### DISCUSSION TITLE
about visualizing neural network.

### DISCUSSION THREAD

**Discussion by ostadnavid** [mrdbourke/pytorch-deep-learning#386] [Q&A]
in 7: 44:00 on model_0. if I wanted to visualize it is it something it looks like? 
![image](https://user-images.githubusercontent.com/121155636/230316064-84950f39-0052-4c71-ba1c-aabb78576543.png)

**Comment by mrdbourke** [comment]
Hi @ostadnavid,

Yes that is correct, most neural networks generally follow the pattern: 

Input layer (takes data in) -> Hidden layer(s) (these can be almost anything + many different combinations) -> Output layer (puts data out)
